<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>基本处理</string>
    </property>
    <addaction name="actionPoygonize"/>
    <addaction name="actionCreatePyramids"/>
    <addaction name="actionSieveFilter"/>
    <addaction name="actionContour"/>
    <addaction name="actionImageAnticolor"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>实例</string>
    </property>
    <addaction name="actionImageResample"/>
    <addaction name="actionImageCut"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionCreatePyramids">
   <property name="text">
    <string>创建金字塔</string>
   </property>
  </action>
  <action name="actionPoygonize">
   <property name="text">
    <string>矢量化</string>
   </property>
  </action>
  <action name="actionSieveFilter">
   <property name="text">
    <string>小斑去除</string>
   </property>
  </action>
  <action name="actionContour">
   <property name="text">
    <string>等高线</string>
   </property>
  </action>
  <action name="actionImageAnticolor">
   <property name="text">
    <string>图像反色</string>
   </property>
  </action>
  <action name="actionImageResample">
   <property name="text">
    <string>图像重采样</string>
   </property>
  </action>
  <action name="actionImageCut">
   <property name="text">
    <string>图像裁剪</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
